pages/common/musescore.md

```diff
@@ -0,0 +1,32 @@
+# musescore
+
+> MuseScore 3 sheet music editor.
+> More information: <https://musescore.org/en/handbook/3/command-line-options>.
+
+- Use a specific audio driver:
+
+`musescore --audio-driver {{jack|alsa|portaudio|pulse}}`
+
+- Set the MP3 output bitrate in kbit/s:
+
+`musescore --bitrate {{bitrate}}`
+
+- Start MuseScore in debug mode:
+
+`musescore --debug`
+
+- Enable experimental features, such as layers:
+
+`musescore --experimental`
+
+- Export the given (or currently opened) file to the specified output file. The file type depends on the extension given:
```

Comment:
"currently opened" &mdash; how does this work with the CLI?

Reply:
At least that's what was at the man page. But when I tested it it doesn't work without an input file. I'll just delete that part